bootstrap.js

```diff
@@ -12,7 +12,12 @@ const { classes: Cc, interfaces: Ci, results: Cr, utils: Cu } = Components;
 
 var Allow = (function () {
 	function wildcard (string) {
-		return new RegExp("^" + string.replace(/\./g, "\\.").replace(/\*/g, ".*?").replace(/\?/g, ".") + "$");
+		// Escape special characters that are special in regular expressions
+		string = string.replace(/(\\|\^|\$|\{|\}|\(|\)|\+|\||\<|\>|\&|\.)/g, "\\$1");
```

Comment:
Actually I purposefully left those out to allow character classes to be specified ([as is possible in many Unix shells](https://en.wikipedia.org/wiki/Wildcard_character#File_and_directory_patterns)). Maybe that should be documented or reconsidered through…